save-backend/src/main/kotlin/org/cqfn/save/backend/repository/BaseEntityRepository.kt

```diff
@@ -1,11 +1,26 @@
 package org.cqfn.save.backend.repository
 
 import org.cqfn.save.entities.BaseEntity
+import org.springframework.data.jpa.domain.Specification
 import org.springframework.data.jpa.repository.JpaRepository
 import org.springframework.data.repository.NoRepositoryBean
+import kotlin.reflect.KClass
 
 /**
  * JPA repositories for the subclasses of the BaseEntity
  */
 @NoRepositoryBean
-interface BaseEntityRepository<T : BaseEntity> : JpaRepository<T, Long>
+interface BaseEntityRepository<T : BaseEntity> : JpaRepository<T, Long> {
+    /**
+     * @param clazz
+     * @return list of class objects
+     */
+    fun getList(clazz: KClass<T>?): List<T>
```

Comment:
What's the purpose of this method? How is it different from `findAll`?

Reply:
You can Specification with any conditions: where, order by etc.

Reply:
Yeah, for the second method it's clear. But this method only accepts a single argument.

Reply:
Yes, removed it